specs/dynamicdependencies/sample-b.1.md

```diff
@@ -0,0 +1,107 @@
+# Sample B.1 - HelloWorld console app using the Boostrap API
+
+At runtime, HelloWorld wants to use the Windows App SDK Framework 1.0-preview1 package, so it calls the Bootstrap API.
+
+## Win32
+
+```c++
+#include <iostream>
+
+#include <MddBootstrap.h>
+
+int main()
+{
+    // Initialize access to Windows App SDK
+    const UINT32 c_majorMinorVersion{ 0x00010000 };
+    PCWSTR c_versionTag{ L"preview1" };
+    const PACKAGE_VERSION c_minVersion{};
+    wprintf(L"MddBootstrapInitialize(0x%08X, \"%s\", %hu.%hu.%hu.%hu)...\n",
+            c_majorMinorVersion, c_versionTag, c_minVersion.Major, c_minVersion.Minor, c_minVersion.Build, c_minVersion.Revision);
+    HRESULT hr{ MddBootstrapInitialize(c_majorMinorVersion, c_versionTag, c_minVersion) };
+    if (FAILED(hr))
+    {
+        wprintf(L"Error 0x%X in MddBootstrapInitialize(0x%08X, \"%s\", %hu.%hu.%hu.%hu)\n",
+                hr, c_majorMinorVersion, c_versionTag,
+                c_minVersion.Major, c_minVersion.Minor, c_minVersion.Build, c_minVersion.Revision);
+        return hr;
+    }
+
+    // Do interesting stuff...
+    std::cout << "Hello World!\n";
+
+    // Cleanup
+    MddBootstrapShutdown();
```

Comment:
Consider showing adding a `wil::unique_call` here that handles calling this shutdown method, even on exceptions.

Reply:
As discussed on the other PR, failure to call shutdown is a minor nit. Is automatic cleanup important? And if so, is that just a C++ thing or should we also provide it for C#?